locales/nl/nl.json

```diff
@@ -4,7 +4,7 @@
     "90 Days": "90 dagen",
     ":amount Total": ":amount Totaal",
     ":resource Details": ":resource Details",
-    ":resource Details: :title": ":resource Details: :title",
+    ":resource Details: :title": "Details: :title",
```

Comment:
Why :resource was deleted ?

Reply:
I think it's : ":resource Details: :title"

Reply:
Wow